tests/distributed/test_workspaces/test_remote_workspaces.py

```diff
@@ -146,6 +146,20 @@ def test_workspace_delete():
         assert client.workspaces.delete(workspace_id)
 
 
+def test_workspace_clear():
+    client = JinaDClient(host=__default_host__, port=8000)
+    workspace_id = client.workspaces.create(
+        paths=[os.path.join(cur_dir, 'empty_flow.yml')]
+    )
+    assert DaemonID(workspace_id).type == 'workspace'
+    assert(
+        WorkspaceItem(**client.workspaces.get(id=workspace_id)).state
+        == RemoteWorkspaceState.ACTIVE
+    )
+    assert workspace_id in client.workspaces.list()
+    assert client.workspaces.delete(workspace_id)
```

Comment:
If I understand correctly, this is testing `client.workspaces.delete`? Would it be possible to add a test to test the invocation of the newly added `clear` method itself?

Reply:
i will work on it and write the correct test for clear method.
Thanks for your inputs